### PR TITLE
CON-2537-Markdown-Average-Calc-exercise

### PR DESCRIPTION
**Pull request by zanninso**

### CODE REVIEW

File: subjects/java/checkpoints/average-calc/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## AverageCalc
+
+### Instructions
+
+Write a function named `average` in a file named `AverageCalc` java. This function takes three parameters: start, end, and step. It returns the average of the numbers between start and end with each number being greater than the previous one by the step given."
```
Comment (nprimo) [inline subjects/java/checkpoints/average-calc/README.md:5]: There is a single `"` at the end of this line

File: subjects/java/checkpoints/average-calc/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+```
+
+### Usage
+
+Here is a possible `ExerciseRunner.java` to test your function :
+
+```java
+public class ExerciseRunner {
+    public static void main(String[] args) {
+        System.out.println(MultiplicationTable.generate(1,5,1));
```
Comment (nprimo) [inline subjects/java/checkpoints/average-calc/README.md:24]: This line is odd: shouldn't the function call be `AverageCalc.average(1,5,1)`?

**Review by nprimo — CHANGES_REQUESTED** [review]

File: subjects/java/checkpoints/average-calc/README.md (hunk 2 above)
Comment (zanninso) [reply subjects/java/checkpoints/average-calc/README.md]: Definitely yeah, it's fixed thanks.

**Review by nprimo — APPROVED** [review]
Subject LGTM now :) 
Solution and tests are missing

**Pull request merged**